apps/sms/views/shared/js/settings.js

```diff
@@ -72,6 +79,15 @@ var Settings = {
     }
   },
 
+  // return resolve promise with sendReadReport status
+  initSendReadReport: function initSendReadReport(value) {
```

Comment:
I wonder what the value is if the setting does not exist yet. Is it undefined ? Should we set a default value if it's undefined ? I think that if we reverse the meaning (disabled instead of enabled), we may want to disable by default and so set the setting to "true".